packages/design-system-twrnc-preset/package.json

```diff
@@ -63,6 +63,7 @@
     "deepmerge": "^4.2.2",
     "jest": "^29.7.0",
     "metro-react-native-babel-preset": "^0.77.0",
+    "react": "^18.2.0",
```

Comment:
Fixing build issue by moving react to devDependencies, not actually sure how it passed CI without it being here 🤔